contactaccessrights/CRM/Contactaccessrights/Helper/ContactActionsMenu/Contact.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * Class CRM_Contactaccessrights_Helper_Contact
+ */
+class CRM_Contactaccessrights_Helper_ContactActionsMenu_Contact {
+
+  /**
+   * Gets the CMS user information for a Civicrm contact
+   *
+   * @param int $contactID
+   *
+   * @return array
+   */
+  public static function getUserInformation($contactID) {
+    $output = [];
+    $output['contact_id'] = $contactID;
+
+    try {
+      $result = civicrm_api3('User', 'getsingle', [
+        'contact_id' => $contactID,
+      ]);
+
+      //When a contact has CMS account, the cmsId parameter is needed for the
+      //HRCore CMSData classes
+      $output['cmsId'] = $result['id'];
+      $output['name'] = $result['name'];
+    } catch(Exception $e) {}
+
+    return $output;
+  }
+
+  /**
+   * Gets the ACL Groups that the contact belongs to.
+   *
+   * @param int $contactID
+   *
+   * @return array
+   */
+  public static function getACLGroups($contactID) {
+    $result = civicrm_api3('GroupContact', 'get', [
+      'sequential' => 1,
+      'contact_id' => $contactID,
+      'api.Group.get' => ['id' => "\$value.group_id", 'sequential' => 1],
+    ]);
+
+    $aclGroups = [];
+
+    if ($result['count'] < 0) {
+      return $aclGroups;
+    }
+
+    $accessControlValue = self::getAccessControlOptionValue();
```

Comment:
just for consistency, wouldn't it make more sense for the variable to also be named `accessControlOptionValue` as the method that is being used to get it?